### PR TITLE
✨ New & Updated Themes (Nord & Rosé Pine)

### PR DESCRIPTION
**Pull request by ehanahamed**
**Changes proposed in this pull request:**

- Updates to Nord Theme (`src/styl/themeNord.styl` & `src/node_requires/monaco-themes/Nord.json`)
- New Rosé Pine Theme (`src/styl/themeRosePine.styl`, `src/styl/themeRosePineMoon.styl`, `src/styl/themeRosePineDawn.styl`, `src/node_requires/monaco-themes/RosePine.json`, `src/node_requires/monaco-themes/RosePineMoon.json`, and `src/node_requires/monaco-themes/RosePineDawn.json`) (based on [rosepinetheme.com](https://rosepinetheme.com))
- Updates to `src/node_requires/themes/index.ts`
- A new directory for 3rd party highlight.js themes not included in the highlight.js node module (`src/node_requires/themes/3rdParty/highlight.js/`)

**Ping @CosmoMyzrailGorynych**

### CODE REVIEW

File: src/styl/themeRosePineDawn.styl [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+accent1 = #d7827e // rose
+error = #b4637a // love
+red = error
+success = #286983 // pine
+green = success
+warning = #ea9d34 // gold
+orange = warning
+
+theme = 'RosePine'
+themeDark = true
```
Comment (CosmoMyzrailGorynych) [inline src/styl/themeRosePineDawn.styl]: This is definitely not a dark theme.

Comment (CosmoMyzrailGorynych) [reply src/styl/themeRosePineDawn.styl]: Also do change theme names in styl files

Comment (ehanahamed) [reply src/styl/themeRosePineDawn.styl]: Oops

**Review by CosmoMyzrailGorynych — CHANGES_REQUESTED** [review]
Besides minor errors, this PR needs an update of src\riotTags\main-menu\license-panel.tag

File: src/node_requires/themes/index.ts [hunk 2]
```diff
@@ -7,13 +7,16 @@ const builtInThemes = [
...
     'PooxelGreen',
     'PooxelBlue',
+    'RosePineDawn',
+    'RosePineMoon',
+    'RosePine',
     'LucasDracula',
     'Night',
-    'HCBlack',
-    'Nord'
+    'HCBlack'
```
Comment (CosmoMyzrailGorynych) [inline src/node_requires/themes/index.ts:19]: Nice

**Review by ehanahamed — COMMENTED** [review]
@CosmoMyzrailGorynych I made all edits and fixes and have updated the license pannel.

**Review by CosmoMyzrailGorynych — APPROVED** [review]

**Comment by CosmoMyzrailGorynych** [comment]
Squash-merged in https://github.com/ct-js/ct-js/commit/a434a184a6ea5b2c8fe2f8d6fed8340624214b93 (had to resolve a semantic merge conflict)

**Comment by CosmoMyzrailGorynych** [comment]
Welp, it will appear as closed on Github. But it is in the trunk now.

**Comment by ehanahamed** [comment]
It seems broken travis ci shall haunt github pull requests forever... 😱 😱

**Pull request closed without merging**